src/app/src/util/constants.js

```diff
@@ -517,3 +517,5 @@ export const PPE_FIELD_NAMES = Object.freeze([
     'ppe_contact_email',
     'ppe_website',
 ]);
+
+export const OARFont = 'ff-tisa-sans-web-pro,sans-serif';
```

Comment:
There are a few other components that reference this font string. Consider spreading the use of this constant to those component to ease the burden on a future maintainer that needs to change the font.

<img width="1096" alt="Screen Shot 2021-05-11 at 2 07 37 PM" src="https://user-images.githubusercontent.com/17363/117885081-3f17de00-b262-11eb-9dc1-f608641a8a83.png">

Reply:
Updated in 5726f0f.